src/components/popupMod.js

```diff
@@ -127,20 +127,17 @@ L.Popup.include({
       let cropTypeSelection = cropType.options[cropType.selectedIndex].text;
       let plantingTime = document.getElementById(`plant_${field_id}`)
       let harvestTime = document.getElementById(`harvest_${field_id}`)
-      if (!plantingTime) {
+      if (plantingTime.value === "" || harvestTime.value === "" ) {
          call_toast(
             "Please ensure that at least\
-            Planting time is selected before saving.",
+            Planting time and Harvest time is selected before saving.",
             5000
          )
-      } else if (plantingTime) {
+      } else {
 
          plantingTime.setAttribute("value", plantingTime.value)
          harvestTime.setAttribute("value", harvestTime.value)
 
-         for (let i = 0; i < cropType.options.length; i++) {
```

Comment:
Does this help solve the wonky croptype dropdown? My browser is in dark mode so I can't tell for sure

Reply:
For me in firefox it prevented the dropdoqn from changing after hitting save
So if a user selected coffee for example it would remain coffee otherwise it would default to Mangoes which is the alst item on the array of cropType